cms-contentful/src/main/java/io/commercetools/sunrise/cms/contentful/ContentfulCmsService.java

```diff
@@ -32,14 +32,20 @@
     @Override
     public CompletionStage<Optional<String>> get(final List<Locale> locales, final CmsIdentifier cmsIdentifier) {
         return getEntry(cmsIdentifier)
-                .thenApplyAsync(cdaEntryOptional -> cdaEntryOptional
+                .thenApply(cdaEntryOptional -> cdaEntryOptional
                         .flatMap(cdaEntry -> getLocalizedField(locales, cdaEntry, cmsIdentifier.getFieldName())));
     }
 
     private CompletionStage<Optional<CDAEntry>> getEntry(final CmsIdentifier cmsIdentifier) {
-        return fetchEntry(cmsIdentifier).thenApply(cdaArray -> Optional.ofNullable(cdaArray)
-                .filter(e -> e.items() != null && !e.items().isEmpty())
-                .map(e -> (CDAEntry) e.items().get(0)));
+        return fetchEntry(cmsIdentifier).handle((cdaArray, error) -> {
+            if (error != null) {
+                return Optional.empty();
```

Comment:
@lauraluiz is it fine: https://github.com/commercetools/commercetools-sunrise-java-cms/pull/2/commits/1dd050355722104c4cb6185a3ac3fad28633aac6?